### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by jmoseka**
In this PR, I have built a mobile web application to check weather updates from a list of the country's capital city.

Home page: is a list of countries 
Detail page: includes the display of weather data from the selected country

The project implements the following objective

- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions and reducers in React.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -13,15 +11,16 @@
   }
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.App {
+  width: 50vw;
```
Comment (prolajumokeoni) [inline src/App.css:15]: - [ ] Based on the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) your web application should be mobile-friendly, please make your design look like the Behance design on mobile you can use the developer tools to see how your website looks on mobile screen sizes when designing 👍🏿  
See video below ⬇️ 

https://user-images.githubusercontent.com/69638013/175778378-8a4442fd-e5b3-4cf2-8593-7e7cc0741eaa.mp4

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @jmoseka,
 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️

- [ ] Beautiful presentation 👏🏿 Your presentation should also contain a highlight about a piece of the code you find  interesting or you are  proud of making based on the [project documentation](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#project-documentation) requirements, please record another video following all the requirements 🤗 

- [ ] Kindly write unit tests for several components of the app and some pure functions in Redux code and  integration tests using React Testing Library
- [ ] Your home page should have a list of categories that could be filtered using given parameters, e.g. category name, you can filter by continents or create a search bar to filter countries alphabetically 😉 
   

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jmoseka** [comment]
- Add test files 
- Repeat video presentation
- Add the search input button
- Responsive to different screen sizes

**Review by DeliceLydia — APPROVED** [review]
## Hi @jmoseka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**